.github/workflows/build-containers.yml

```diff
@@ -0,0 +1,69 @@
+name: Build SPRAS Containers
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
I checked options at https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request and these three look good. I'm still not sure what synchronize is even after looking at the docs.

Reply:
Agreed, they do a poor job of documenting it, but it basically means "whenever someone updates the PR":
https://github.com/orgs/community/discussions/24567

Reply:
I removed the input in a recent commit.